Justfile

```diff
@@ -144,6 +144,7 @@ run-graph-builder:
 		address = "127.0.0.1"
 		port = 8080
 		path_prefix = "{{path_prefix}}"
+		tracing_endpoint = "localhost:6831"
```

Comment:
you could add a global variable e.g. `default_tracing_endpoint` and then reuse that throughout the Justfile